lisa/tests/base.py

```diff
@@ -466,6 +469,75 @@ def test_noisy_tasks(self, noise_threshold_pct=None, noise_threshold_ms=None):
         res.add_metric("noisiest task", metric)
 
         return res
+
+    @classmethod
+    #pylint: disable=unused-argument
+    def check_noisy_tasks(cls, noise_threshold_pct=None, noise_threshold_ms=None):
+        """
+        Decorator that applies :meth:`test_noisy_tasks` to the underlying function
+
+        We also expose :meth:`test_noisy_tasks` parameters to the decorated
+        function.
+        """
+        def decorator(func):
+            @cls.test_noisy_tasks.used_events
+            @functools.wraps(func)
+            def wrapper(self, *args,
+                        noise_threshold_pct=noise_threshold_pct,
+                        noise_threshold_ms=noise_threshold_ms,
+                        **kwargs):
+                res = func(self, *args, **kwargs)
+
+                print(noise_threshold_pct)
```

Comment:
remove debug print or turn it into a `cls.get_logger().XXX` call

Reply:
Woops!